Mathlib/Tactic/NormNum/Core.lean

```diff
@@ -299,18 +311,19 @@ def derive {α : Q(Type u)} (e : Q($α)) (post := false) : MetaM (Result e) := d
     let lit : Q(ℕ) := e
     return .isNat (q(instAddMonoidWithOneNat) : Q(AddMonoidWithOne ℕ))
       lit (q(IsNat.raw_refl $lit) : Expr)
-  let s ← saveState
-  let arr ← (normNumExt.getState (← getEnv)).2.getMatch e
-  for ext in arr do
-    if (bif post then ext.post else ext.pre) then
-      try
-        let new ← ext.eval e
-        trace[Tactic.norm_num] "{e} ==> {new}"
-        return new
-      catch err =>
-        trace[Tactic.norm_num] "{e} failed: {err.toMessageData}"
-        s.restore
-  throwError "{e}: no norm_nums apply"
+  profileitM Exception "norm_num" (← getOptions) do
```

Comment:
Do you want to leave this in place?

Reply:
I do. In theory, this should make `norm_num` show up in `set_option profile true` reports; `simp` is doing the same thing and we should put these markers on other heavy tactics that people might want to profile.